fees/mevx.ts

```diff
@@ -1,6 +1,70 @@
-import { FetchOptions, SimpleAdapter } from "../adapters/types";
+import { Chain } from "@defillama/sdk/build/general"
+import { Adapter, ChainBlocks, FetchOptions, FetchResultFees, } from "../adapters/types";
 import { CHAIN } from "../helpers/chains";
 import { getSolanaReceived } from "../helpers/token";
+import { queryFlipside } from "../helpers/flipsidecrypto";
+
+interface IVolume {
+  amount: number;
+  tokenAddress: string;
+  tx: string;
+}
+
+type TAddress = {
+  [s: string | Chain]: string;
+}
+
+const address: TAddress = {
+  [CHAIN.ETHEREUM]: '0x087297b9987F16Ee251137b59D001aCf2457579e',
+  [CHAIN.BASE]: '0x764B099A48187D5f27806C6739BDa0BEF90B69F9',
+  [CHAIN.BSC]: '0xfA90769c3D13Feab848bfBC06A99a8B86B981dC6',
+
+}
+
+const graph = (chain: Chain) => {
+  return async ({ createBalances, getFromBlock, getToBlock, }: FetchOptions): Promise<FetchResultFees> => {
+
+    const query = `
+        select
+          input_data,
+          TX_HASH
+        from
+          ${chain}.core.fact_transactions
+        WHERE
+        BLOCK_NUMBER > ${await getFromBlock()} AND BLOCK_NUMBER < ${await getToBlock()}
+        and to_address = '${address[chain]}'
+        and status = 'SUCCESS'
+      `
+
+
+    const value: string[][] = (await queryFlipside(query, 260))
+    const rawData = value.map((a: string[]) => {
+      const data = a[0].replace('0x5f575529', '');

```

Comment:
sorry for the delayed review, but can you verify the contracts on the explorer or add abi here, so we can parse the input? Also, are these contracts still used? I see that the calls more or less stopped 2 weeks ago